src/pages/Lending/Pool/components/Borrow/BorrowInput.tsx

```diff
@@ -96,6 +99,32 @@ export const BorrowInput = ({
   const [fromAddress, setFromAddress] = useState<string | null>(null)
   const [borrowAssetIsFiat, toggleBorrowAssetIsFiat] = useToggle(false)
   const [collateralAssetIsFiat, toggleCollateralAssetIsFiat] = useToggle(false)
+  const [quoteSlippageDecimalPercentage, setQuoteSlippageDecimalPercentage] = useState<
+    number | null
+  >()
+  const [isUnsafeQuoteNoticeDismissed, setIsUnsafeQuoteNoticeDismissed] = useState<boolean | null>(
+    null,
+  )
+
+  const unsafeSlippageDecimalPercent = 0.05 // 5%
```

Comment:
Do we want to extract this outside of component scope as `UNSAFE_SLIPPAGE_DECIMAL_PERCENT`?

Reply:
We should! Actioned in https://github.com/shapeshift/web/pull/6569/commits/6b2888c1d04597f110cd4fddbed0ee38867bc2c8.